app/abilities/sbv/group_ability.rb

```diff
@@ -0,0 +1,9 @@
+module Sbv::GroupAbility
+  extend ActiveSupport::Concern
+
+  included do
+    on(Group) do
+      permission(:any).may(:'index_event/festivals').all
```

Comment:
I would like to have a better name for this. This one seems to be the automatically generated one, but I would (at the cost of having to check it manually) prefer a name like `:index_festival_events` to be more in line with the other abilities.

Reply:
well, actually, it should stay this way. I see a way to clean this up, but it would involve touching all other wagons that have special permission like that. maybe I will tackle this some other time, for now, I would leave like it is now.